pkg/supportbundle/rbac.go

```diff
@@ -34,34 +34,75 @@ func checkRemoteCollectorRBAC(ctx context.Context, clientConfig *rest.Config, ti
 
 	var forbidden []error
 
-	spec := authorizationv1.SelfSubjectAccessReviewSpec{
-		ResourceAttributes: &authorizationv1.ResourceAttributes{
-			Namespace:   namespace,
-			Verb:        "create,delete",
-			Group:       "",
-			Version:     "",
-			Resource:    "pods,configmap",
-			Subresource: "",
-			Name:        "",
+	resourceAttributesList := []authorizationv1.ResourceAttributes{
+		{
+			Namespace: namespace,
```

Comment:
should we check on empty `namespace`?

Reply:
Is there a use case where you think we might need to? Remote host collection can execute within a single namespace right?

Reply:
I just think passing empty `namespace` should not allowed as we are checking RBAC in both cluster-scope and namespace-scoped such as `get/pods`

Reply:
Are you saying we should default to `default` for the namespace or fail if no namespace is provided? Right now, if the user doesn't provide a namespace when executing remote host collectors via the CLI, then we pretty much assume `""` for the namespace. 

This may be a larger thing we want to address in the project as I've noticed that the namespace in the kubeconfig context isn't correctly used everywhere. This would be another one of those issues that's a larger thing we need to revisit in the project as the behavior is the same for in-cluster collectors today. 

Let me know if that clears things up, or maybe i'm still not quite following what you're suggesting.